### PR TITLE
Psp-3229 create a research file via link

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

**Comment by asanchezr** [comment]
PR is failing with following error

image

File: frontend/src/store/rootReducer.ts [hunk 1]
```diff
@@ -22,11 +20,9 @@ export const reducer = {
   [systemConstantsSlice.name]: systemConstantsSlice.reducer,
   [networkSlice.name]: networkSlice.reducer,
   [leafletMouseSlice.name]: leafletMouseSlice.reducer,
-  [parcelLayerDataSlice.name]: parcelLayerDataSlice.reducer,
```
Comment (FuriousLlama) [inline frontend/src/store/rootReducer.ts:25]: nice cleanup! thanks!

File: frontend/src/hooks/useDraftMarkerSynchronizer.ts [hunk 2]
```diff
@@ -66,14 +65,16 @@ const useDraftMarkerSynchronizer = (nameSpace: string) => {
...
     (values: any, initialValues: any, nameSpace: string) => {
-      const draftMarkers = getDraftMarkers(values, initialValues, nameSpace);
-      if (draftMarkers.length) {
-        dispatch(storeDraftProperties(draftMarkers as PointFeature[]));
-      } else {
-        dispatch(storeDraftProperties([]));
+      if (isMounted()) {
+        const draftMarkers = getDraftMarkers(values, initialValues, nameSpace);
+        if (draftMarkers.length) {
+          setDraftProperties(draftMarkers as PointFeature[]);
```
[the remark this answers is not in the document]
Comment (devinleighsmith) [reply frontend/src/hooks/useDraftMarkerSynchronizer.ts:71]: I'll need to have a look and see if that is possible. Basically anything with a lat/lng works here, so restricting to IProperty may not make sense.

File: frontend/src/features/properties/selector/components/MapClickMonitor.tsx [hunk 3]
```diff
@@ -14,33 +17,37 @@ interface IMapClickMonitorProps {
...
+    .getCenter();
+  return new PropertyForm({
+    pid: selectedFeature?.properties?.PID_NUMBER ?? '',
+    pin: selectedFeature?.properties?.PIN ?? '',
+    latitude: selectedFeature?.properties?.CLICK_LAT_LNG?.lat ?? latLng.lat ?? '',
+    longitude: selectedFeature?.properties?.CLICK_LAT_LNG?.lng ?? latLng.lng ?? '',
+    planNumber: selectedFeature?.properties?.PLAN_NUMBER ?? '',
+    address: 'placeholder', //todo: need alternate source for this
+    legalDescription: 'placeholder', //todo: need access to fully attributed parcelmap bc layer,
+    district: selectedFeature?.properties?.REGIONAL_DISTRICT ?? '', // todo: this returns a named district,
```
[the remark this answers is not in the document]
Comment (devinleighsmith) [reply frontend/src/features/properties/selector/components/MapClickMonitor.tsx:49]: Yeah, I think that should be possible, will confirm.

[the remark this answers is not in the document]
Comment (devinleighsmith) [reply frontend/src/features/properties/selector/components/MapClickMonitor.tsx:49]: worth noting that all I'm doing here is moving some code from a previous sprint, but I don't mind updating this to change the type.

**Review by FuriousLlama — APPROVED** [review]

**Pull request merged**